src/networking.c

```diff
@@ -3181,6 +3181,16 @@ size_t getClientMemoryUsage(client *c, size_t *output_buffer_mem_usage) {
     mem += c->argv_len_sum;
     mem += multiStateMemOverhead(c);
 
+    /* Add memory overhead of pubsub channels and patterns. Note: this is just the overhead of the robj pointers
+     * the strings themselves because they aren't stored per client */
```

Comment:
```suggestion
     * the strings themselves are not counted because they aren't stored per client */
```

Reply:
Thanks for finding the typo, committed a fix.